assets/components/bundle/bundle.jsx

```diff
@@ -35,7 +33,6 @@ export default function Bundle(props: PropTypes) {
     <div className={className}>
       <div style={{ float: 'right' }}>
         {props.showPaymentLogos ? <InlinePaymentLogos /> : null}
```

Comment:
You could actually ditch the wrapping div and add `float: 'right`' to the style for `InlinePaymentLogos`, because I made the opposite change for the test